evaluation.py

```diff
@@ -89,11 +88,13 @@ def save_stacked_evaluation_heatmaps(self, X, y, filename):
         stacked_predict_heatmaps=self.stacked_predict_heatmaps(predict_heatmaps)
         stacked_ground_truth_heatmaps=self.stacked_ground_truth_heatmaps(X, y)
         
-        # Reshape heatmaps to 3 channels, normalize channels to [0,255]
-        stacked_predict_heatmaps = cv2.cvtColor(stacked_predict_heatmaps, cv2.COLOR_GRAY2RGB)
-        stacked_predict_heatmaps = cv2.normalize(stacked_predict_heatmaps, None, alpha=0, beta=255.0, norm_type=cv2.NORM_MINMAX, dtype=cv2.CV_32F)
+        # Reshape heatmaps to 3 channels with colour injection, normalize channels to [0,255]
+        # TODO - normalize individual heatmaps rather than a whole stack at once.
```

Comment:
This looks great! Re: normalizing individual heat maps, we might not want to do that because it'll just scale heat maps where it predicts nothing (very small values) to everything is lit up. we should set max across an entire hg layer or across the entire model imo

Reply:
Good thinking. No change necessary then